translations/nl-be.json

```diff
@@ -1182,8 +1184,25 @@
   "all-content": "Alle inhoud",
   "news-items": "Kort bestek berichten",
   "document-not-on-agendaitem": "Niet geüpload in agendapunt",
+  "export-as-pdf": "Exporteren als PDF",
+  "concerns": "Betreft",
+  "update-thing": "{thing} updaten",
+  "current-version": "Huidige versie",
+  "monday": "maandag",
+  "tuesday": "dinsdag",
+  "wednesday": "woensdag",
+  "thursday": "donderdag",
+  "friday": "vrijdag",
+  "saturday": "zaterdag",
+  "sunday": "zondag",
```

Comment:
Not sure, but I think `date-fns` has a way to display the weekday name according to the app's locale (`nl-be`). Maybe look into that to see if its the case? https://date-fns.org/

Reply:
implemented in [5dbecf7](https://github.com/kanselarij-vlaanderen/frontend-kaleidos/pull/1713/commits/5dbecf7cb44370e6f771d6cff79e0ab931be8377)